logstash-core/src/main/java/org/logstash/plugins/aliases/AliasPlugin.java

```diff
@@ -0,0 +1,61 @@
+package org.logstash.plugins.aliases;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * A POJO class to map AliasRegistry.yml structure.
+ */
+public class AliasPlugin {
```

Comment:
I think the usage of `AliasPlugin` is to retrieve information and not expose the ability to change it, so it has mainly an immutable state after creation. To respect this I would remove all the setters a make the assignment to final fields into the constructor.

Reply:
Thanks @andsel for pointing out. Forgot to remove `set` methods after my IDEA generated :)